app/views/endemics/select-the-herd.njk

```diff
@@ -0,0 +1,142 @@
+{% extends './layouts/layout.njk' %}
+
+{% set isSingleHerd = herds.length == 1 %}
+
+{% if isSingleHerd %}
+  {% set pageTitleText = "Is this the same "+herdOrFlock+" you have previously claimed for?" %}
+{% else %}
+  {% set pageTitleText = "Select the "+herdOrFlock+" you are claiming for" %}
+{% endif %}
```

Comment:
Does it make sense for this logic to be in the page, or would it be better placed in the API layer?